solutions/usaco-815.mdx

```diff
@@ -7,6 +7,10 @@ author: Shijie Ren
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_taming_gold_feb18.html)
 
+This solution is $\mathcal{O}(N^2)$ which is faster than the official editorial's.
+
+But not required since $N \leq 100$.
```

Comment:
grammar kinda sus here

Reply:
I'm sorry but I am really bad at writing

Reply:
dw about it